### DISCUSSION TITLE
Reset filters

### DISCUSSION THREAD

**Discussion by Kugeleis** [widgetti/solara#661] [Q&A] [answered]
Is there a simple way to reset all filters?
I use multiple filters like `CrossFilterSelect` and `CrossFilterSlider` on a DataFrame and would like to reset (delete) the filters to the initial state. Is there an easy way?

**Comment by maartenbreddels** [comment, marked as answer]
I think that kind of is a missing feature. I managed to get something a workaround working:
https://app.py.cafe/maartenbreddels/solara-crossfilter-reset-workaround

Let me know if you think that is sufficient
